packages/lesswrong/lib/sql/LogTableQuery.ts

```diff
@@ -0,0 +1,19 @@
+import Query from "./Query";
+import Table from "./Table";
+
+/**
+ * Builds a Postgres query to alter a table to be LOGGED (ie to use the Write-Ahead Log).
+ * This is the default, so this query is only useful for reverting a table to the default.
+ */
+
+class LogTableQuery<T extends DbObject> extends Query<T> {
+  constructor(table: Table<T>) {
+    super(table, [
+      "ALTER TABLE",
+      table,
```

Comment:
`table` needs to be double-quoted to be case-sensitive. You can either do that manually, or just add it as an atom and it should work correctly, like:
```
super(table, ["ALTER TABLE"]);
this.atoms.push(table);
this.atoms.push("SET LOGGED");
```

The same applies in `UnlogTableQuery` too.